File: from-1001-to-1100/1051-height-checker/solution.py
class Solution(object):
    def heightChecker(self, heights):
        """
        :type heights: List[int]
        :rtype: int
        """
        if not heights:
            return 0
        
        hc = heights[:]
        hc.sort()
        
        pi = 0
        pc = 0
        result = 0
        
        while pi < len(heights):
            if heights[pi] != hc[pi]:
                result += 1
            pi += 1
            
        return result
        

File: from-1001-to-1100/1051-height-checker/solution.go
func heightChecker(heights []int) int {
    expected := make([]int, len(heights))
    copy(expected, heights)
    sort.Ints(expected)
    result := 0
    for i := 0; i < len(heights); i ++ {
        if heights[i] != expected[i] {
            result += 1
        }
    }
    return result
}